src/validations/basic/schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Validation",
+  "definitions": {
+    "Validation": {
+      "title": "Basic validation",
+      "type": "object",
+      "properties": {
+        "minScore": {
+          "title": "Minimum score",
+          "type": "number",
+          "minimum": 1
+        },
+        "useVotingStrategies": {
+          "description": "Use voting strategies",
+          "type": "boolean",
+          "default": false
+        },
+        "strategies": {
+          "title": "Strategies",
+          "type": "array",
+          "items": {
+            "title": "Strategy",
+            "type": "object",
+            "default": {
+              "name": "ticket",
+              "network": "1",
+              "params": {
+                "symbol": "DAI"
+              }
+            },
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "network": {
+                "type": "string"
+              },
+              "params": {
+                "type": "object"
+              }
+            },
+            "required": ["name", "params"]
+          }
+        }
+      },
+      "required": ["minScore"],
```

Comment:
strategies are also required no? 😅

Reply:
Not anymore since we have the changes here https://github.com/snapshot-labs/snapshot/pull/3775